packages/contentstack-import/src/import/modules-js/webhooks.js

```diff
@@ -98,6 +98,15 @@ module.exports = class ImportWebhooks {
         .catch(function (error) {
           // error while importing environments
           fileHelper.writeFileSync(webFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).webhook !== undefined) {
+            if (JSON.parse(error.request.data).webhook.name) {
+              message_content_type =
+                ' Failed webhook: ' + JSON.parse(error.request.data).webhook.name;
+            }
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code